app/services/oregon_digital/itis_service.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module OregonDigital
+  class ItisService < OddEndpointService
+    def self.fetch_itis_statement(vocabulary, subject)
+      statement(subject, itis_label(vocabulary, subject))
+    end
+
+    private
+
+    def self.itis_label(vocabulary, subject)
+      parse_json(vocabulary.as_query(subject.to_s))
+    end
+
+    def self.parse_json(query)
+      json_parse_service.json(query)['scientificName']['combinedName']
```

Comment:
This might as well return the full json representation of the query result.
Push up the responsibility of finding the label to `itis_label`